src/dotnet-roslyn-tools/CreateReleaseTags/CreateReleaseTags.cs

```diff
@@ -47,43 +48,43 @@ public static async Task<int> CreateReleaseTagsAsync(RoslynToolsSettings setting
 
             logger.LogInformation("Tagging releases...");
 
-            foreach (var visualStudioRelease in visualStudioReleases)
+            foreach (var visualStudioRelease in visualStudioReleases.Reverse())
```

Comment:
This might need a comment for why the .Reverse() is there.

Reply:
oh, this was actually just for my debugging.. it was annoying waiting for it to spin through all of the versions that weren't going to be relevant, to find one that was so I could actually see my code working :)